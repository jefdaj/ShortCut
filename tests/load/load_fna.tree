.
|-- cache
|   `-- load
|       |-- 737dfdb34e.fna
|       `-- 737dfdb34e.fna.ids
|-- exprs
|   |-- any
|   |   `-- 387993be85.fna.list -> ../../cache/lines/...
|   |-- list
|   |   |-- 0e506f60ac.fna.list -> ../../cache/lines/...
|   |   |-- 556607931c.str.list -> ../../cache/lines/...
|   |   |-- 5730090ccf.fna.list -> ../../cache/lines/...
|   |   `-- 777e3c6c61.fna.list.list -> ../../cache/lines/...
|   |-- load_fna
|   |   `-- 34a32d8e2f.fna -> ../../cache/load/737dfdb34e.fna
|   `-- str
|       `-- a2fd369878.str -> ../../cache/lines/...
`-- vars
    |-- mapped.fna.list -> ../exprs/list/5730090ccf.fna.list
    |-- result -> ../exprs/any/387993be85.fna.list
    `-- single.fna -> ../exprs/load_fna/34a32d8e2f.fna

8 directories, 12 files
