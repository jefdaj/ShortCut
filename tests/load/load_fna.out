[>BAA16579.2
ATGATCTCCACTACCTCCCTGTTTGCCCCCGTTGACCAAGACCTCCGTTTATTAACGGATAATCTCAAGCGGCTTGT...
>BAA16580.1
GTGGGGCGTTTGGATCAAGATAGCGAAGGACTATTGCTGCTCACCAGCAACGGTAAACTTCAGCATCGTTTGGCCCA...
>BAA16581.1
...
]
